doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.md

```diff
@@ -0,0 +1,173 @@
+(kuberay-raycluster-quickstart)=
+
+# RayCluster Quickstart
+
+This guide shows you how to manage and interact with Ray clusters on Kubernetes.
+
+## Preparation
+
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) (>= 1.23), [Helm](https://helm.sh/docs/intro/install/) (>= v3.4) if needed, [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), and [Docker](https://docs.docker.com/engine/install/).
+* Make sure your Kubernetes cluster has at least 4 CPU and 4 GB RAM.
+
+## Step 1: Create a Kubernetes cluster
+
+This step creates a local Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/). If you already have a Kubernetes cluster, you can skip this step.
+
+```sh
+kind create cluster --image=kindest/node:v1.26.0
+```
+
+## Step 2: Deploy a KubeRay operator
+
+Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator from the Helm repository.
+
+(raycluster-deploy)=
+## Step 3: Deploy a RayCluster custom resource
+
+Once the KubeRay operator is running, you're ready to deploy a RayCluster. Create a RayCluster Custom Resource (CR) in the `default` namespace.
+
+```sh
+# Deploy a sample RayCluster CR from the KubeRay Helm chart repo:
+helm install raycluster kuberay/ray-cluster --version 1.4.0
+```
+
+Use `helm install raycluster kuberay/ray-cluster --version 1.4.0 --set 'image.tag=2.41.0-aarch64'` instead if are using ARM64 (Apple Silicon) machines.
```

Comment:
KubeRay v1.4.0 uses Ray 2.46.0 which supports multi-arch, so we no longer need this comment.

```suggestion
```